react/features/transcription/components/TranscriptionSubtitles.web.js

```diff
@@ -284,29 +82,16 @@ class TranscriptionSubtitles extends React.Component {
      * @returns {ReactElement}
      */
     render() {
-        const paragraphs = [];
-
-        this.state.knownIDs.forEach(id => {
-            const o = this.state[id];
-            let text;
-
-            if (o) {
-                text = `${o.name}: `;
-
-                if (o.final) {
-                    text += o.final;
-                } else {
-                    const stable = o.stable ? o.stable : '';
-                    const unstable = o.unstable ? o.unstable : '';
-
-                    text += stable + unstable;
-                }
-            }
-
-            paragraphs.push(<p key = { id }> { text } </p>);
-        });
+        if (this.state.hidden) {
```

Comment:
we should probably create a mechanism for setting this somehow before we merge the PR

Reply:
As in an option in UI to enable/disable subtitles ?